src/interval/interval.rs

```diff
@@ -2,43 +2,68 @@ use crate::interval::errors::IntervalError;
 use crate::note::{Note, PitchClass};
 use strum_macros::Display;
 
+/// The quality of an interval; major, minor, etc.
 #[derive(Display, Debug, Copy, Clone)]
 pub enum Quality {
+    /// A perfect interval; unisons, fourths, fifths, and octaves.
     Perfect,
+    /// A major interval.
     Major,
+    /// A minor interval.
     Minor,
+    /// An augmented interval.
     Augmented,
+    /// A diminished interval.
     Diminished,
 }
 
+/// The number of an interval.
 #[derive(Display, Debug, Copy, Clone)]
 pub enum Number {
+    /// The unison interval (the same note).
     Unison,
+    /// The second interval.
```

Comment:
Please remove the comments that just copies the enum name and doesn't add any value from this line on.